packages/sign-client/src/controllers/engine.ts

```diff
@@ -748,11 +1163,12 @@ export class Engine extends IEngine {
     await this.client.core.history.resolve(payload);
   };
 
-  private sendError: EnginePrivate["sendError"] = async (id, topic, error) => {
+  private sendError: EnginePrivate["sendError"] = async (params) => {
+    const { id, topic, error, encodeOpts } = params;
```

Comment:
`encodeOpts` sounds like a function.
```suggestion
    const { id, topic, error, encodedOpts } = params;
```

Reply:
I would leave it as is because this is param for `encode` fx and imo seems intuitive when you go over the code